dangerzone/gui/__init__.py

```diff
@@ -71,18 +71,6 @@ def gui_main(custom_container, filename):
     # Allow Ctrl-C to smoothly quit the program instead of throwing an exception
     signal.signal(signal.SIGINT, signal.SIG_DFL)
 
-    # If we're using Linux and docker, see if we need to add the user to the docker group or if the user prefers typing their password
-    if platform.system() == "Linux":
-        if not gui_common.ensure_docker_group_preference():
-            return
-        try:
-            if not gui_common.ensure_docker_service_is_started():
-                click.echo("Failed to start docker service")
-                return
-        except AuthorizationFailed:
-            click.echo("Authorization failed")
-            return
-
     # See if we need to install Docker...
```

Comment:
So for MacOS you still need/install Docker? (If so, then the doc, i.e. `BUILD.md` etc. should probably reflect this.)

Or does this also need to be removed?

Reply:
macOS and Windows will still need docker. Whether it's docker or podman, these are all Linux containers, and if you're not using Linux you'll need a Linux VM to run the containers in. I'm hoping to be able to ship the macOS version with a VM of its own to remove the Docker Desktop dependency though (#118)

Reply:
Add instructions :) https://github.com/firstlookmedia/dangerzone/pull/120/commits/9536a4db41bf0e584f510b88a15cf215ad9ea446